tf_files/configs/kube-setup-workvm.sh

```diff
@@ -86,25 +89,25 @@ fi
 EOF
   fi
 
-  if ! grep 'kubectl completion bash' ${WORKSPACE}/.bashrc > /dev/null; then 
-    cat - >>${WORKSPACE}/.bashrc << EOF
+  if ! grep 'kubectl completion bash' ${WORKSPACE}/.${RC_FILE} > /dev/null; then 
+    cat - >>${WORKSPACE}/.${RC_FILE} << EOF
 if which kubectl > /dev/null 2>&1; then
   # Load the kubectl completion code for bash into the current shell
-  source <(kubectl completion bash)
+  source <(kubectl completion ${SHELLL})
```

Comment:
SHELLLLLLL typo

Reply:
SHELLL is not a typo, Linux already uses SHELL to identify the current shell, didn't want to override that env var. I can rename it to something else and less prone to confusion.

Reply:
CURRENT_SHELL is used now, looks better.